packages/staking/src/features/store/delegationPortfolioStore/stateMachine/__tests__/normalizePercentages.test.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-magic-numbers */
+import { normalizePercentages } from '../normalizePercentages';
+
+const thirds = [{ value: 100 / 3 }, { value: 100 / 3 }, { value: 100 / 3 }];
+const sevenths = Array.from({ length: 7 }, () => ({ value: 100 / 7 }));
+const key = 'value';
+
+describe('Testing normalizePercentages function', () => {
+  test('Should correctly normalize numbers to 0 decimal places', () => {
```

Comment:
A nitpick, but the word `should` is redundant 😉 
```suggestion
  test('correctly normalizes numbers to 0 decimal places', () => {
```